### ISSUE TITLE
Compatibility with next@11.1.0

### ISSUE THREAD

**Issue by dferber90** [toomuchdesign/next-page-tester#245]
I'm getting the following error with `next-page-tester@0.27.2` and `next@11.1.0`

```
 FAIL   xxx  tests/imprint.test.tsx
  ● Test suite failed to run

    Cannot find module 'next/dist/next-server/server/config' from '../../node_modules/next-page-tester/dist/nextConfig.js'

    Require stack:
      /Users/xxx/node_modules/next-page-tester/dist/nextConfig.js
      /Users/xxx/node_modules/next-page-tester/dist/utils.js
      /Users/xxx/node_modules/next-page-tester/dist/getPageObject/makeRouteInfo.js
      /Users/xxx/node_modules/next-page-tester/dist/getPageObject/index.js
      /Users/xxx/node_modules/next-page-tester/dist/page/getPageInfo.js
      /Users/xxx/node_modules/next-page-tester/dist/page/index.js
      /Users/xxx/node_modules/next-page-tester/dist/getPage.js
      /Users/xxx/node_modules/next-page-tester/dist/pure.js
      /Users/xxx/node_modules/next-page-tester/dist/index.js
      tests/getPages.ts
      tests/imprint.test.tsx

      at Resolver.resolveModule (../../node_modules/jest-resolve/build/resolver.js:311:11)
      at Object.<anonymous> (../../node_modules/next-page-tester/dist/nextConfig.js:7:34)
```

It seems like the `next/dist/next-server/server/config` file has changed to `next/dist/server/config` so the path needs to be updated:

I found these occurrences but there might be more:

https://github.com/toomuchdesign/next-page-tester/blob/f3ad5cd777e08790009a7c0820f07a65c26d5b3b/src/nextConfig.ts#L1

https://github.com/toomuchdesign/next-page-tester/blob/f3ad5cd777e08790009a7c0820f07a65c26d5b3b/src/nextConfig.ts#L15

**Comment by toomuchdesign** [comment]
Next.js v11.1 support should available with v0.28. Thanks for raising the issue 👍

**Comment by dferber90** [comment]
That's great to hear! Thanks for handling it so fast! 🙏
